5.1.thats_the_way.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+
+def that_is_the_way(path):
+    """
+    The function that_is_the_way takes a path as an argument and returns
+     a list of files in that path that contain the substring 'deep'.
+
+    :param path: path of directory to search in
+    :return: list of matches
+    """
+    return [file for file in os.listdir(path) if 'deep' in file]
```

Comment:
Your function also returns file that does not start with deep, but contain it in the file-name

Reply:
I didn't understand the the comments.

Reply:
We only want to get files that **start** with the word "deep", so for example a file named "George likes to double-deep" should not be in the return list of this function.